sdk/core/System.ClientModel/tests/Pipeline/ClientPipelineFunctionalTests.cs

```diff
@@ -211,7 +211,7 @@ public void TimesOutResponseBuffering()
         message.BufferResponse = true;
 
         var exception = Assert.ThrowsAsync<TaskCanceledException>(async () => await pipeline.SendSyncOrAsync(message, IsAsync));
-        Assert.AreEqual("The operation was cancelled because it exceeded the configured timeout of 0:00:00.5. ", exception!.Message);
+        Assert.AreEqual("The operation was cancelled because it exceeded the configured timeout of 0:00:00.5. The default timeout can be adjusted by passing a custom ClientPipelineOptions.NetworkTimeout value to the client's constructor. See https://aka.ms/net/scm/configure/networktimeout for more information.", exception!.Message);
```

Comment:
I wonder if we should extract the error message to an internal readonly mask so that the tests don't have to take a hardcoded assumption in the message text.

Reply:
I've done it both ways, but don't have strong feelings one way or the other. I'm going to leave it as is for now since it is more consistent with other tests.